### DISCUSSION TITLE
VSOMEIP Monthly Meeting

### DISCUSSION THREAD

**Discussion by paulboyes** [COVESA/vsomeip#503] [Announcements]
Hi,

At the request of some Members, I have created a monthly VSOMEIP meeting starting August 21st at 7am PT / 10am ET / 16:00 CET.   The purpose of the meeting is to address all things [VSOMEIP](https://github.com/COVESA/vsomeip) including issues, questions, direction, etc… as well as reduce fragmentation.  

You can subscribe to the [Community Calendar by following the instructions on the wiki ](https://wiki.covesa.global/display/WIK4/COVESA+Common+Meeting+Schedule)or to this event via link below.

https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=Mm42Z3V1c2J0YnJxYW1uc2Y3ZHUwaTdlbTRfMjAyMzA4MjFUMTQwMDAwWiBjXzdkMjhjZHU4cm50Y2huYzFmdGxqa2R0OXNjQGc&tmsrc=c_7d28cdu8rntchnc1ftljkdt9sc%40group.calendar.google.com&scp=ALL

Please share this email with your co-workers and colleagues that you would like to attend.

Let me know if you have any questions.

Thank you,

Paul Boyes
Community Director

**Comment by DiogoPedrozza** [comment]
Hi @paulboyes .

The vSomeIP development team will not be able to join the meeting at the proposed date (August 21st).
Please note that due to vacations the team is severely reduced and does not have the time to attend such meeting, our suggestion is that the meeting is postponed to the last 2 weeks of September, when the team is back to the normal capacity.

Also upon internal discussion we believe that a regular meeting every 6 weeks (instead of the proposed monthly) would be better for both parts.

Sorry for any inconvenience caused.
Best regards,
Diogo Pedrosa

CC: @goncaloalmeida, @fcmonteiro, @BrunoLSilvaCTW, @lutzbichler

**Reply by NagarjunaReddyG5262** [reply]
Hi Can you please let me know when there will be a meeting on vsomeip. Can you please share the calender specific to vsomeip?

**Reply by fcmonteiro** [reply]
Next meeting will be Monday (19Feb) at 16.00CET
You can check the calendar here: https://wiki.covesa.global/display/WIK4/COVESA+Common+Meeting+Schedule
The calendar also contains the link to the zoom meeting.
